CHANGELOG.md

```
@@ -142,7 +174,11 @@
 - quickstart.py 対話形式セットアップウィザード
 - CLI コマンド（fetch, status, monitor, init）
 
-[Unreleased]: https://github.com/miyamamoto/jrvltsql/compare/v1.4.0...HEAD
+[Unreleased]: https://github.com/miyamamoto/jrvltsql/compare/v1.6.1...HEAD
+[1.6.1]: https://github.com/miyamamoto/jrvltsql/compare/v1.6.0...v1.6.1
+[1.6.0]: https://github.com/miyamamoto/jrvltsql/releases/tag/v1.6.0
+[1.5.0]: https://github.com/miyamamoto/jrvltsql/releases/tag/v1.5.0
+[1.4.1]: https://github.com/miyamamoto/jrvltsql/releases/tag/v1.4.1
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "1.4.1"

(MD053, link-image-reference-definitions)